### DISCUSSION TITLE
`createStore` is not exported by `zustand/vanilla`

### DISCUSSION THREAD

**Discussion by lvnam96** [pmndrs/zustand#1772] [General] [answered]
I'm trying to use zustand in my Svelte project following your doc [here](https://docs.pmnd.rs/zustand/recipes/recipes#using-zustand-without-react).
I have checked your code to make sure [`createStore` is exported](https://github.com/pmndrs/zustand/blob/9062ca634fa9c2e14ee5005bd37c0f087ae99b7f/src/vanilla.ts#L105).

As you can see in [demo](https://svelte.dev/repl/73ddea22022948cbbaa65c0493126bd6), it throws this error:
```
'createStore' is not exported by https://unpkg.com/zustand@4.3.7/vanilla.js, imported by ./App.svelte
```

If I try `import create from 'zustand/vanilla'`, the compiler does not throw error.

AFAIK, default export is [deprecated](https://github.com/pmndrs/zustand/blob/9062ca634fa9c2e14ee5005bd37c0f087ae99b7f/src/vanilla.ts#L114). What should I use?

**Comment by dai-shi** [comment, marked as answer]
https://unpkg.com/browse/zustand@4.3.7/vanilla.js
It does export `createStore` but it may trick something.

https://unpkg.com/browse/zustand@4.3.7/esm/vanilla.js
This is ESM, so it should work better. I'm not sure how Svelte chooses "exports" in package.json. Does it have any way to tell to use ESM?

**Comment by lvnam96** [comment]
Tried on another REPL & it does work. I have no idea why it happens in svelte REPL.

Thanks for quick support!
